<launch>
    <arg name="marker_to_root_pose" default="[3, 2]"/>
    <node pkg="tf" type="static_transform_publisher" name="mock_world" args="-0.2 0.0 0.0 0 0.0 0.0 ar_marker_0 world 1" />
    <node pkg="sr_camera_pose_estimation" type="cam_tf_pub.py" name="tmp" output="screen">
        <rosparam param="marker_to_root_pose" subst_value="True">$(arg marker_to_root_pose)</rosparam>
    </node>

    <include file="$(find sr_camera_launch)/launch/kinect.launch" />
    <include file="$(find sr_marker_recognition)/launch/alvar.launch" />
</launch>